app/assets/javascripts/discourse/app/templates/preferences.hbs

```diff
@@ -1,5 +1,5 @@
 {{#d-section pageClass="user-preferences" class="user-secondary-navigation"}}
-  {{#mobile-nav class="preferences-nav" desktopClass="preferences-list action-list nav-stacked" currentPath=router._router.currentPath}}
+  {{#mobile-nav class="preferences-nav" desktopClass="preferences-list action-list nav-stacked" currentPath="blah"}}
```

Comment:
is blah expected here ?

Reply:
Yes of course, `blah` is a very important route name 😛 

Will fix, thanks